### PR TITLE
Finalversion2

### PR DESCRIPTION
**Pull request by Ninjaphil24**

### CODE REVIEW

**Comment by Ninjaphil24** [comment]
Dear Reviewer,

Whilst merging a different branch from a different pull request, the branch for my finalversion was merged as well, by accident. I tried reverting, but it didn't do anything useful, or at least I don't know how to make it do what it needs to do.

Please look at the reviewer's comments from the first review on this link:
#3

Concerning the reviewer's comments I have done the following:
About the Requested Changes:
I have added some instructions to the README file, as requested. I must note here that this comes into contradiction with a previous review, where said instructions were considered superfluous and the reviewer asked that I remove them. Between the two, I agree with the more laconic approach, as telling someone how to upload a simple html file, seems superfluous.

Concerning the OPTIONL requests I have done the following:

I have removed all blank lines that I could find.
I have removed my comments, even though those comments were not ununsed code, but comments to help me see where a large div ended. This is something that I will continue to practice in future, but today, for the purpse of the review I have removed them.
I have removed actual unused code that I had forgotten to remove.
On the matter of creating a class for properties, this is an excellent piece of advice. I will implement this in future itterations of the project and create this practice. As it is an OPTIONAL request, for the purposes of this week's submission I will not make the alterations today, as it may take too long and I'll fall behind for the submission. However, I fully intend to do this soon.
I have repaired the margin for the 1st See Project button on page 2.
I will look into centering the cards better, however, as it is an OPTIONAL request, for the purposes of this week's submission I will not make the alterations today, as it may take too long and I'll fall behind for the submission. However, I fully intend to do this soon.
For the overflow problem in the languages/frameworks/skills section, I have repaired it by changing the mediaquery value. This seems to make it work quite well for now, but if there is to be a tablet version of the site, I will perhaps create one more media query.
I have decreased the font-size in the mobile version for the media buttons at the very end. Since I have personalised the site for myself, I only use three of the five platforms suggested by the template, and therefore feel that I shouldn't make any other alterations. Also for the desktop version, I feel that the size should be as big as it is, to create aesthetic balance.
I hope we're good and can submit the project now.

Best,
Philip

File: README.md [hunk 1]
```diff
@@ -1,15 +1,31 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Developers Portfolio of Philip Modinos
+# Developer's Portfolio of Philip Modinos
 
 > This is the professional developer portfolio of Philip Modinos.  Philip has been coding since 2019, focusing on html/css and php/mysql using XAMPP.  He has developed several sites for German music companies "Opera Classica Europa" and "Bad Ems Music Festival" including the site www.offenbachgp.com, which was an online singing competition, that included applause and comments by the audience and a fully automated registration system.  
+The current webpage is fully responsive and has several live links, leading to other existing projects.  For the inactive links, there will be an update later in time, once those projects have been reuploaded.  
 
-![screenshot](./Screenshot_1.png)
+![screenshot](./readmedemo.png)
```
Comment (moise10r) [inline README.md]: - [ OPTIONAL] There is an issue with the screenshot which does not display, kindly fix that

File: index.css [hunk 2]
```diff
@@ -462,11 +442,8 @@ header li a:hover {
 }
 
 .page3 .container .nested {
```
Comment (moise10r) [inline index.css:444]: - [OPTIANAL ] As required by the previous review, you can decrease the font size of the social media icons as well as some margin and padding, You must replicate the design as much as possible (e.g., font, colors, images, tests, margins) I'm approving the project but please fix this before merging.
> 
> Your design:
> 
> 
> 
> Figma design:
> 
> 
>

**Review by moise10r — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @Whoistolu** , 

### **Congratulations! 🎉 Project Approved ✅**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Cheers and Happy coding!👏👏👏💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**